go/src/koding/klient/machine/machine.go

```diff
@@ -29,6 +30,13 @@ type ID string
 // IDSlice represents a set of machine IDs.
 type IDSlice []ID
 
+// Sort sorts machine IDs in lexicographical order.
+func (ids IDSlice) Sort() {
+	sort.Slice(ids, func(i, j int) bool {
+		return string(ids[i]) < string(ids[j])
+	})
+}
+
```

Comment:
Wouldn't it be better to have just `func (IDSlice) Less(i, j int) bool` and then do `sort.Slice(ids, ids.Less)` externally? You wouldn't need to alloc a func each time. Just an idea.

Reply:
I can change this. Just for clarification - I don't allocate a new slice here. I'm reusing existing one.